src/database.cpp

```diff
@@ -184,3 +184,15 @@ void Database::removeLoc(QString locname) {
     if (!query.exec())
         qDebug() << "ERROR: " << query.lastError().text();
 }
+
+int Database::AvaliableSeat(QString BusName, QString date, QString time) {
```

Comment:
busname, date and time is not enough to identify a ticket uniquely. A person can go from Rangpur to Dhaka in one bus, another person may return from Dhaka to Rangpur using the same bus. So please add, "from" and "to" parameters